src/lbaf/Utils/lbsJSONDataFilesMaker.py

```diff
@@ -0,0 +1,569 @@
+"""
+Utility to create and export a data set supporting shared blocks by using a specification file.
+
+To call this script either call script with
+- `lbaf-vt-data-files-maker` if lbaf package is installed or
+- `python src/lbaf/Utils/lbsJSONDatasetMaker.py`
```

Comment:
The name seems to be `lbsJSONDataFilesMaker.py`

Reply:
Oh sorry you are right ! It has been fixed !